CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and **Merged pull requests**. Critical items to know are:
 Referenced versions in headers are tagged on Github, in parentheses are for pypi.
 
 ## [vxx](https://github.com/urlstechie/urlschecker-python/tree/master) (master)
+ - accelerate code using asyncio and aiohttp (0.0.23)
```

Comment:
This should be a version bump,up to 0.1.0 since we would be fundamentally changing the core library and making it un-usable for older python versions.